core/src/main/scala/org/apache/spark/internal/config/package.scala

```diff
@@ -630,6 +636,12 @@ package object config {
       .doubleConf
       .createWithDefault(1.0)
 
+  private[spark] val DYN_ALLOCATION_EXECUTOR_DEALLOCATION_RATIO =
+    ConfigBuilder("spark.dynamicAllocation.executorDeallocationRatio")
```

Comment:
If this only applies to streaming dra we should put streaming in the name.. add docs

Reply:
renaming these configs